splash/tests/lua_modules/emulation.lua

```diff
@@ -102,4 +102,19 @@ function emulation.render_png(splash)
 end
 
 
+function emulation.render_jpeg(splash)
+    splash:go_and_wait(splash.args)
+    splash:set_result_content_type("image/jpeg")
+    local render_all = (splash.args.render_all or
+            splash.args.viewport == "full")
+    return splash:jpeg{
+        width=splash.args.width,
+        height=splash.args.height,
+        render_all=render_all,
+        scale_method=splash.args.scale_method,
+        quality=splash.args.quality,
+    }
+end
```

Comment:
It seems this function is unused in tests. Could you please add a test to test_execute_emulation?

Proxy interface also look untested; to enable it add a test case to test_proxy_server (no need to test with use_gzip=True).